README.md

```diff
@@ -1,93 +1,95 @@
 # solid-doodle
 Repo for our first Hack Day in December 2019.
- 
-## Hacker Name
-Lucifer †   
+ ___
+## Hacker Name : **Lucifer †**
 
-## Description
-A free video calling application without CAPEX and maintainance using agora.io
+### Description
+A free video calling application without CAPEX and maintainance involved for Android.
 
-## Link
+### Link
 https://github.com/amannirala13/FreeEye
 
-## Hacker Name 
-Alphons Jaimon
+___
+
+## Hacker Name : Alphons Jaimon
 
-## Description
+### Description
 Learning to use Azure Cloud Cognitive services and GitHub integration. Mood Detector - Captures your face then detects the mood like Happy or Sad
 
-## Link
+### Link
 https://github.com/AJV009/mooddetector/
 
-## Team Name
-HackPac
+___
+
+## Team Name : HackPac
 
-## Team Members
+### Team Members
 - [Vishesh Luthra](http://guthub.com/visheshluthra)
 - [Disha Patel](http://guthub.com/disha110)
 - [Nandini Garg](http://guthub.com/nandinigarg21)
 
-## Description
+### Description
 Amrita's mother has stricty asked her not to waste any paper. Her BFF Natasha is visiting her after a long time. Can you help Amrita with a TIC TAC TOE app?This is a digitised version of a classic game of TIC TAC TOE. Use of this app will reduce the wastage of paper and make it easier for the user to keep a score.
 
-## Link
+### Link
 https://github.com/visheshluthra/improved-eureka
 
+___
 
-## Team Name
-New Coders
+## Team Name : New Coders
 
-## Team Members
+### Team Members
 - Meghna Das
 - Simeen Khan Ayyub
 - Shritama Sengupta
 
-## Description
+#### Description
```

Comment:
There's an extra # here. Can you please fix it?

Reply:
Fixed the issue in line 46.